lib/ansible/modules/cloud/ovirt/ovirt_clusters.py

```diff
@@ -495,7 +511,8 @@ def update_check(self, entity):
                 sorted([
                     str(source) for source in entity.required_rng_sources
                 ])
-            )
+            ) and
+            equal(getattr(self._get_mac_pool(), 'id', None), getattr(entity.mac_pool, 'id', None))
```

Comment:
Mac pool is always exposed, no? So no need to `getattr(entity.mac_pool, 'id', None)` but `entity.mac_pool.id` should be enough?